packages/engines/core-engine/src/game-engine/engines/lorcana/src/cards/lorcana-card-instance.ts

```diff
@@ -94,13 +95,27 @@ export class LorcanaCardInstance extends CoreCardInstance<
     return true;
   }
 
+  get type(): LorcanaCardDefinition["type"] {
+    return this.card.type;
+  }
+
   get meta(): LorcanaCardMeta {
     const ctx = this.contextProvider.getCtx();
-    return (ctx.cardMetas?.[this.instanceId] as LorcanaCardMeta) || {};
+    const meta = ctx.cardMetas?.[this.instanceId];
+    return (meta || {}) as LorcanaCardMeta;
   }
 
-  get type(): LorcanaCardDefinition["type"] {
-    return this.card.type;
+  get strength(): number {
+    const baseStrength = (this.card as any).strength || 0;
```

Comment:
please don't use `any` as a type, investigatee why this.card doesn't have the approapriate type. To the modifications